src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1852,6 +1852,69 @@ limitations under the License.
             <include cluster="Mode Select" client="false" server="true" clientLocked="false" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-room-airconditioner</name>
+        <domain>HA</domain>
+        <typeName>Matter Room Air Conditioner</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0072</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+                <requireCommand>IdentifyQuery</requireCommand>
```

Comment:
This does not make sense.  This command is explicitly _disallowed_ in Matter.  Yes, I know other device types here probably get it wrong too; please file an issue on them and don't make the problem worse.